src/main/java/nom/tam/image/compression/hdu/CompressedImageHDU.java

```diff
@@ -142,6 +144,48 @@ public CompressedImageHDU(Header hdr, CompressedImageData datum) {
     }
 
     public ImageHDU asImageHDU() throws FitsException {
+        final Header header = getGenericHeader();
+        ImageData data = (ImageData) ImageHDU.manufactureData(header);
+        ImageHDU imageHDU = new ImageHDU(header, data);
+        data.setBuffer(getUncompressedData());
+        return imageHDU;
+    }
+
+     /**
+     * Given this compressed HDU, get the original (decompressed) axes.
+     *
+     * @return the dimensions of the axis.
+     * @throws FitsException
+     *             if the axis are configured wrong.
+     */
+    public int[] getDecompressedAxes() throws FitsException {
```

Comment:
I wonder if this would be better named as `getImageAxes()`. To me that would make it a bit more intuitive as to what it represents...